### PR TITLE
Add Tech and Support for BME680 

### PR DESCRIPTION
**Pull request by mtaylorsherwood**
Add me in as a new technician for the project and apply bronze level.

Fix a small typo in the readme.

Add support for BME680 sensor.

### CODE REVIEW

File: app/app.py [hunk 1]
```diff
@@ -25,6 +25,8 @@
         sensor = growbme280()
     if sensor_type == "bmp280":
         sensor = growbmp280()
+    if sensor_type == "bme680":
+        sensor = growbme680(config["sensors"]["oversample"])
```
Comment (alexellis) [inline app/app.py:29]: Let's just have that as the default and make it less complicated?

**Comment by alexellis** [comment]
Thanks for the interest. I've added a note.

We will also need this for the data logger. Don't worry that it's duplicated.

Alex

**Review by alexellis — CHANGES_REQUESTED** [review]
Submitting a review

**Comment by mtaylorsherwood** [comment]
Thanks for the feedback. Have added the sensor to the data logger as well as removing that config option for simplicity.

**Comment by alexellis** [comment]
@pensarguet can you take a look and compare to your own patch?

**Comment by mtaylorsherwood** [comment]
> @pensarguet can you take a look and compare to your own patch?

I'm happy to refractor mine and tidy it up once their PR is merged in, to be honest thr history is a bit messy now

**Comment by pensarguet** [comment]
Hi Alex, 

The code from Martyn is more complete than mine and embed the data logger and the calibration stage before getting the value.
For the logic, it's similar ;-)

**Comment by mtaylorsherwood** [comment]
Closing so other PR's can be done and I can rebase

**Pull request closed without merging**